docs/en/api/options.md

```diff
@@ -54,12 +54,33 @@ const wrapper = shallow(Component, {
   slots: {
     default: [Foo, Bar],
     fooBar: Foo, // Will match `<slot name="FooBar" />`.
-    foo: '<div />'
+    foo: '<div />',
+    bar: 'bar'
   }
 })
 expect(wrapper.find('div')).toBe(true)
 ```
 
+#### Passing text
+
+You can pass text to `slots`.  
+There are two limitations to this.
+
+This works at vue@2.2+.
+
+This works for the text as below.
+
+```js
+const wrapper = mount(ComponentWithSlots, { slots: { default: 'foobar' }})
+```
+
+This does not work for the text as below.
```

Comment:
This does not work for the text as below.

>

This does not work for the text below.